### DISCUSSION TITLE
In Star Trek The Next Generation, artificial character Data has a cat named what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#86] [Galaxy Trivia] [answered]
**Hint:** Boston Dynamics 🤖🐶

**Comment by oscarxdev** [comment, marked as answer]
In Star Trek: The Next Generation, the artificial character Data has a cat named "Spot". Spot is a female cat who first appeared in the episode "Data's Day" in season 4 of the show. Spot is a recurring character throughout the series and is known for being somewhat temperamental and difficult to handle. Despite this, Data is very fond of Spot and takes great care in her upbringing. Spot is also notable for giving birth to a litter of kittens, much to Data's surprise and confusion.
IA HUHA

**Reply by ParanoidUser** [reply]
> In Star Trek: The Next Generation, the artificial character Data has a cat named "Spot". Spot is a female cat who first appeared in the episode "Data's Day" in season 4 of the show. Spot is a recurring character throughout the series and is known for being somewhat temperamental and difficult to handle. Despite this, Data is very fond of Spot and takes great care in her upbringing. Spot is also notable for giving birth to a litter of kittens, much to Data's surprise and confusion. IA HUHA

Hey @NetJox4!

Check to see if you have marked up answers to your own questions somewhere (in other repositories as well). This may prevent you from earning the *Galaxy Brain* badge. Learn more - https://github.com/orgs/community/discussions/18454

**Reply by oscarxdev** [reply]
Hey @ParanoidUser 
I just checked the article but i think i already did it all 😌
